kong/runloop/handler.lua

```diff
@@ -690,6 +691,7 @@ return {
     before = function(ctx)
       local var = ngx.var
       local header = ngx.header
+      local cookie_hash_data = ctx.balancer_hash_cookie
```

Comment:
It seems like there is no point in retrieving this value outside of the `ctx.KONG_PROXIED` branch. It would also be more readable if we retrieved it just above the `if cookie_hash_data` branch created below.

Reply:
Additionally, this gives the impression that it belongs to the `ngx.ctx.balancer_address` structure, in which we already store all balancer-related values for decision making and introspection purposes (its name should be revisited already). Doing so also allows us to reduce usage of the top-level `ctx` table (which is sensitive to naming conflicts), and self-documentation by adding this field to the ones documented with its [initialization code](https://github.com/Kong/kong/blob/next/kong/runloop/handler.lua#L525-L534). cc @Tieske @hishamhm